ajax/libs/primeng/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "primeng",
+  "filename": "primeng.min.js",
+  "description": "PrimeNG is a collection of rich UI components for Angular 2",
+  "version": "2.0.1",
+  "keywords": [
+    "angular",
+    "ui",
+    "components",
+    "datatable",
+    "datagrid",
+    "charts",
+    "typescript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/primefaces/primeng.git"
+  },
+  "homepage": "http://www.primefaces.org/primeng/",
+  "author": "cagataycivici",
+  "license": "MIT",
+  "npmName": "primeng",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "primeng.js*",
+        "resources/**/!(*.scss)"
```

Comment:
@extend1994 I discuss with @pvnr0082t about this, and I think we should not only fetch only two themes' `css`.

@pvnr0082t Would you give some advice about this situation? 
And please help me review on this PR. Thanks.